src/core/Akka.MultiNodeTestRunner/Program.cs

```diff
@@ -132,7 +132,7 @@ static void Main(string[] args)
                             continue;
                         }
 
-                        if (!string.IsNullOrWhiteSpace(specName) && !test.Value.First().MethodName.Contains(specName))
+                        if (!string.IsNullOrWhiteSpace(specName) && !test.Value[0].MethodName.Contains(specName))
```

Comment:
Added this change to make it easier to debug with watches when I'm launching the MNTR